blackspots/blackspots.json

```
@@ -0,0 +1,68 @@
+{
+  "type": "dataset",
+  "id": "blackspots",
+  "title": "blackspots",
+  "status": "beschikbaar",
+  "description": "Blackspots",
+  "version": "0.0.1",
+  "crs": "EPSG:28992",
+  "tables": [
+    {
+      "id": "blackspots",
+      "type": "table",
+      "schema": {
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "type": "object",
+        "additionalProperties": false,
+        "required": [
+          "schema",
+          "id",
+          "locatie id",
+          "spot type",
+          "geometry",
+          "stadsdeel",
+          "status"
+        ],
+        "display": "locatie id",
+        "properties": {
+          "schema": {
+            "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
+          },
+          "id": {
+            "type": "integer"
+          },
+          "locatie id": {
+            "type": "string"
+          },
+          "spot type": {
+            "type": "string"
+          },
+          "geometry": {
+            "$ref": "https://geojson.org/schema/Point.json"
+          },
+          "stadsdeel": {
+            "type": "string"
+          },
+          "status": {
+            "type": "string"
+          },
+          "start uitvoering": {
+            "type": "string"
+          },
+          "eind uitvoering": {
+            "type": "string"
+          },
+          "jaar blackspotlijst": {
+            "type": "integer"
```

Ik zou hier graag op de een of andere manier een type-hint aan mee willen geven dat het om een jaar gaat.

Ik zat te denken aan een reference aan een 'year' veld definitie in het meta schema maar dat is ook weer een hoop gedoe en werkt ook weer niet lekker met andere definities van jaar (wat als je een min/max op wil geven in je eigen veld.)

Wellicht willen we er de UCUM unit voor Gregoriaans jaar (a_g) aan meegeven? 

Niet de mooiste unit maar het zou wel werken... tenzij we nog iets netters kunnen bedenken...

```suggestion
            "type": "integer",
            "unit": "a_g"
```

---

Lijkt me een prima oplossing, heb ik opgenomen.